src/index.js

```diff
@@ -74,6 +77,9 @@ var types = {
     },
 
     integer: function (s) {
+    if(registerTypes.hasOwnProperty('integer')){
```

Comment:
Looking at these functions and all the replication it makes sense to abstract it to a function and just pass the name in.